sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;//this is for vertically centering arrows in ::after
+  height: 30px;
```

Comment:
For the record, I was ok with pixels for these since we're matching Susan's exact sizing here and it makes sense to not use math, not use variables that tie the sizes to anything else (although we could add a new variable to [our UI component list](https://github.com/cb-talent-development/employer-style-base/blob/topic/EM1091-homepage-wrap-up/sass/variables/_sizing.scss#L31)), etc. I was also okay with converting the pixels to relative sizes so that they do scale if someone zooms or has a different font size setting on their browser so it's good that you've done that